configs/crowddet/crowddet-rcnn_r50_fpn_8xb2-30e_crowdhuman.py

```diff
@@ -0,0 +1,208 @@
+        loss_cls=dict(type='CrossEntropyLoss', loss_weight=1.0),
+        loss_bbox=dict(type='L1Loss', loss_weight=1.0)),
+    roi_head=dict(
+        type='MultiInstanceRoIHead',
+        bbox_roi_extractor=dict(
+            type='SingleRoIExtractor',
+            roi_layer=dict(
+                type='RoIAlign',
+                output_size=7,
+                sampling_ratio=-1,
+                aligned=True,
+                use_torchvision=True),
+            out_channels=256,
+            featmap_strides=[4, 8, 16, 32]),
+        bbox_head=dict(
+            type='MultiInstanceBBoxHead',
+            with_refine=False,
+            num_shared_fcs=2,
+            in_channels=256,
+            fc_out_channels=1024,
+            roi_feat_size=7,
+            num_classes=1,
+            bbox_coder=dict(
+                type='DeltaXYWHBBoxCoder',
+                target_means=[0., 0., 0., 0.],
+                target_stds=[0.1, 0.1, 0.2, 0.2]),
+            reg_class_agnostic=False,
+            loss_cls=dict(
+                type='CrossEntropyLoss',
+                loss_weight=1.0,
+                use_sigmoid=False,
+                reduction='none'),
+            loss_bbox=dict(
+                type='SmoothL1Loss', loss_weight=1.0, reduction='none'))),
+    # model training and testing settings
+    train_cfg=dict(
+        rpn=dict(
+            assigner=dict(
+                type='MaxIoUAssigner',
+                pos_iou_thr=0.7,
+                neg_iou_thr=(0.3, 0.7),
+                min_pos_iou=0.3,
+                match_low_quality=True,
+                ignore_iof_thr=-1),
+            sampler=dict(
+                type='RandomSampler',
+                num=256,
+                pos_fraction=0.5,
+                neg_pos_ub=-1,
+                add_gt_as_proposals=False),
+            allowed_border=-1,
+            pos_weight=-1,
+            debug=False),
+        rpn_proposal=dict(
+            nms_pre=2400,
+            max_per_img=2000,
+            nms=dict(type='nms', iou_threshold=0.7),
+            min_bbox_size=2),
+        rcnn=dict(
+            assigner=dict(
+                type='MultiInstanceAssigner',
+                pos_iou_thr=0.5,
+                neg_iou_thr=0.5,
+                min_pos_iou=0.3,
+                match_low_quality=False,
+                ignore_iof_thr=-1),
+            sampler=dict(
+                type='MultiInsRandomSampler',
+                num=512,
+                pos_fraction=0.5,
+                neg_pos_ub=-1,
+                add_gt_as_proposals=False),
+            pos_weight=-1,
+            debug=False)),
+    test_cfg=dict(
+        rpn=dict(
+            nms_pre=1200,
+            max_per_img=1000,
+            nms=dict(type='nms', iou_threshold=0.7),
+            min_bbox_size=2),
+        rcnn=dict(
+            nms=dict(type='nms', iou_threshold=0.5),
+            score_thr=0.01,
+            max_per_img=500)))
+
+dataset_type = 'CrowdHumanDataset'
```

Comment:
why not create a crowd_human_detection.py in `_base_/datasets`?

Reply:
The setting of crowd_human in CrowdDet is not general. It requires a unique DataPreprocessor.